### PR TITLE
DEV-15884: Allow adding Shipments for already marked as send OrderUnits

### PR DESCRIPTION
**Pull request by dariusbepunkt**
Reopened the PR due to accidental revert.

### CODE REVIEW

**Comment by dariusbepunkt** [comment]
@aazon Now it is up to date again.

File: src/Endpoints/Shipments/Post.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+use Hitmeister\Component\Api\Transfers\OrderUnitShipmentTransfer;
+
+/**
+ * @category PHP-SDK
+ * @package  Hitmeister\Component\Api\Endpoints\Shipments
+ * @author   Darius Brückers <darius.brueckers@real-digital.de>
+ * @license  https://opensource.org/licenses/MIT MIT
+ * @link     https://www.hitmeister.de/api/v1/
+ */
+class Post extends AbstractEndpoint implements IdAware
```
Comment (johnholiver) [inline src/Endpoints/Shipments/Post.php]: Shouldn't implement IdAware

File: src/Endpoints/Shipments/Post.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	 */
+	public function setTransfer(OrderUnitShipmentTransfer $transfer)
+	{
+		$this->transfer = $transfer;
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	protected function getUriPattern()
```
Comment (johnholiver) [inline src/Endpoints/Shipments/Post.php]: `getUriPattern` -> `getURI`

File: src/Namespaces/ShipmentsNamespace.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	{
+		$data = new OrderUnitShipmentTransfer();
+		$data->id_order_unit = $orderUnitId;
+		$shipmentInformation = new ShipmentInformationTransfer();
+		$shipmentInformation->carrier_code = $carrierCode;
+		$shipmentInformation->tracking_number = $trackingNumber;
+
+		$endpoint = new Post($this->getTransport());
+		$endpoint->setTransfer($data);
+		$endpoint->setId($orderUnitId);
```
Comment (johnholiver) [inline src/Namespaces/ShipmentsNamespace.php]: This line must be removed as you dont' do `/shipments/{id}`

**Comment by johnholiver** [comment]
You also need to edit `CHANGELOG.md` and `src/Client.php to` to update the version

**Pull request merged**